modelon/impact/client/entities.py

```diff
@@ -1996,6 +2053,43 @@ def keys(self):
         return self._variables
 
 
+class ExternalResult:
+    """
+    Class containing  external result.
+    """
+
+    def __init__(self, result_id, workspace_service=None):
+        self._result_id = result_id
+        self._workspace_sal = workspace_service
+
+    def __repr__(self):
+        return f"Result id '{self._result_id}'"
+
+    def __eq__(self, obj):
+        return isinstance(obj, ExternalResult) and obj._result_id == self._result_id
+
+    @property
+    def id(self):
+        """Result id"""
+        return self._result_id
+
+    @property
+    def metadata(self):
+        """External result metadata."""
+        upload_meta = self._workspace_sal.get_uploaded_result_meta(self._result_id)[
+            "data"
+        ]
+        id = upload_meta.get("id")
+        name = upload_meta.get("name")
+        created_at = upload_meta.get("createdAt")
```

Comment:
If we want to expose the date we should create a datetime object. Here is a stackoverflow on how to convert the isoformat string: https://stackoverflow.com/questions/127803/how-do-i-parse-an-iso-8601-formatted-date. So we would need to install the python-dateutil package to have it work with Python 3.6.

Or we could just remove it for now.

Reply:
I have removed it for now. Thanks! 😄